conf/drupal/config/core.entity_form_display.node.news.default.yml

```diff
@@ -250,9 +251,16 @@ content:
     weight: 32
     region: content
     settings:
-      rows: 5
+      rows: 20
       placeholder: ''
-    third_party_settings: {  }
+    third_party_settings:
+      allowed_formats:
```

Comment:
@joeg8612  this config change does not related to task description. Was that added on purpose?  
If yes - should you update this to the ticket description and add that to change log? 
Or it was just added automatically when you exported the configs?

Reply:
After investigation and meeting with @joeg8612 , we figured out that this config change is related to the Content (field_news_body) and will not affect the form itself, until `maxlength_js` is `null`. I suppose that config was exported since `maxlength` was installed after `core.entity_form_display.node.news.default.yml` was create. Number of rows was changed intentionally.